tests/python/test_bit_struct.py

```diff
@@ -31,6 +31,10 @@ def verify_val():
     set_val()
     verify_val()
 
+    # test bit_struct SNode read and write in python scope
```

Comment:
```suggestion
    # Test bit_struct SNode read and write in Python-scope by calling the wrapped, untranslated function body
```

I like this trick :-)